xds/src/main/java/io/grpc/xds/EdsLoadBalancer.java

```diff
@@ -188,7 +203,7 @@ XdsClient createXdsClient() {
 
   @Override
   public void handleNameResolutionError(Status error) {
-    channelLogger.log(ChannelLogLevel.ERROR, "Name resolution error: {0}", error);
+    logger.log(XdsLogLevel.WARNING, "Received name resolution error: {0}", error);
```

Comment:
nit: add `\n` after `:`

Reply:
New line here won't help much. The string format of `Status` will still be something like `[... cause=<STACKTRACE>]`.